--- src/core/lib/iomgr/ev_epollex_linux.cc      2023-01-17 21:35:00.895729004 -0500
+++ src/core/lib/iomgr/ev_epollex_linux.cc      2023-01-17 21:35:00.899728958 -0500
@@ -1146,7 +1146,7 @@
 }

 #ifndef NDEBUG
-static long gettid(void) { return syscall(__NR_gettid); }
+static long sys_gettid(void) { return syscall(__NR_gettid); }
 #endif

 /* pollset->mu lock must be held by the caller before calling this.
@@ -1166,7 +1166,7 @@
 #define WORKER_PTR (&worker)
 #endif
 #ifndef NDEBUG
-  WORKER_PTR->originator = gettid();
+  WORKER_PTR->originator = sys_gettid();
 #endif
   if (grpc_polling_trace.enabled()) {
     gpr_log(GPR_INFO,
